bundled/tool/server.py

```diff
@@ -30,13 +30,11 @@ def update_sys_path(path_to_add: str, strategy: str) -> None:
     os.getenv("LS_IMPORT_STRATEGY", "useBundled"),
 )
 
+
 # **********************************************************
 # Imports needed for the language server goes below this.
 # **********************************************************
 # pylint: disable=wrong-import-position,import-error
-import jsonrpc
-import utils
-from pygls import lsp, protocol, server, uris, workspace
```

Comment:
You will have to restore these, in the same location.

Reply:
Updates made. I didn't see where I could add the Labels for the linter though.